build/BosGenaDrop/src/Mintbase/App/Solutions/Index.jsx

```diff
@@ -0,0 +1,62 @@
+const accountId = props.accountId ?? "bos.genadrop.near";
+
+const mode = props.mode || Storage.get("mode") || "light";
+
+const customStyle = `
+  width:100% !important;
+  background: rgba(40,42,58,0.05);
+  padding: 5px 0;
+    ul {
+    justify-content: flex-start!important;
+    background: none !important;
+    width: max-content !important;
+    margin-bottom: 0;
+    li {
+        font-weight: 500;
+        margin: 0 10px;    
+    }
+    }
+  @media (max-width: 500px) {
+    font-size: 12px;
+  }
+`;
+
+const [activeTabIndex, setActiveTabIndex] = useState(0);
+const [activeRangeIndex, setActiveRangeIndex] = useState(0);
+
+const handleTabClick = (index) => {
+  setActiveTabIndex(index);
+};
+
+const handleRangeClick = (index) => {
+  setActiveRangeIndex(index);
+};
+
+return(
+    <>
+     <Widget
+          src={`${accountId}/widget/Mintbase.App.Navbar.Index`}
+          props={{ mode }}
+    />
+    <Widget
+         src={`${accountId}/widget/Mintbase.MbTabs`}
+        props={{
+          tabLabels: ["Enterprise", "Top Affiliates", "Use Cases"],
+          mode,
```

Comment:
for small tabs like this, i think you should build a custom Router in this component, because now on Click every single time, the widget loads again, you can reference what I did in the explore. I'm gonna optimize this component so that when you import it as a function that bug is fixed

Reply:
What can i are need fix? Can you explain more?